stdlib/2and3/keyword.pyi

```diff
@@ -1,6 +1,9 @@
-# Stubs for keyword
-
+import sys
 from typing import Sequence, Text, Union
 
-def iskeyword(s: Union[Text, bytes]) -> bool: ...
+def iskeyword(s: Union[Text, str]) -> bool: ...
 kwlist: Sequence[str]
+
+if sys.version_info >= (3, 9):
+    def issoftkeyword(s: str) -> bool: ...
+    softkwlist: Sequence[str]
```

Comment:
Both are undocumented at the moment.

Reply:
Actually, both are not present in 3.9.0b3, are they more recent than that?

Reply:
yeah, it's very new: https://github.com/python/cpython/pull/20877#issuecomment-643874877